debian/amd64/jessie/entry.sh

```diff
@@ -58,7 +58,9 @@ function init_systemd()
 {
 	GREEN='\033[0;32m'
 	echo -e "${GREEN}Systemd init system enabled."
-	env > /etc/docker.env
+	for var in $(compgen -e); do
+    	printf '%q="%q"\n' "$var" "${!var}"
```

Comment:
all oft these should be indented one tab level in. consider it approved once this is fixed